### ISSUE TITLE
Have an harder personal handicap

### ISSUE THREAD

**Issue by c-geek** [duniter/duniter#868]
Today the handicap is defined as:

    PERSONAL_HANDICAP = FLOOR(LN(1 + PERSONAL_EXCESS) / LN(1.189))

So this is a purely linear difficulty augmentation: if our computer is 16 times stronger than the co-calculators, our difficulty will go through:

* `PERSONAL_EXCESS = 1` => 18% harder than normally
* `PERSONAL_EXCESS = 2` => 41% harder than normally
* `PERSONAL_EXCESS = 3` => 68% harder than normally
* `PERSONAL_EXCESS = 4` => 100% harder than normally

With this algorithm, the difficulty increases with a delay, and may never handicap us to our have a difficulty corresponding to our effective superiority.

To reach it more quickly, we could simply use an exponential:

    PERSONAL_HANDICAP = FLOOR(LN(1 + PERSONAL_EXCESS) / LN(1.189))*2

The result would be:

* `PERSONAL_EXCESS = 1` => 41% harder than normally
* `PERSONAL_EXCESS = 2` => 100% harder than normally
* `PERSONAL_EXCESS = 3` => 182% harder than normally
* `PERSONAL_EXCESS = 4` => 300% harder than normally

Have to study a bit more this before release.

**Comment by jytou** [comment]
Is this calculated "per member"? What if I have a raspberry that is on 24/7 and a computer that calculates, let's say, only at night?

**Comment by c-geek** [comment]
Yes per member, you can have as many computers as you want, the difficulty is the same for all of them.
